app/controllers/users/sessions_controller.rb

```diff
@@ -308,6 +311,10 @@ def randomize_check_password?
       SecureRandom.random_number(IdentityConfig.store.compromised_password_randomizer_value) >=
         IdentityConfig.store.compromised_password_randomizer_threshold
     end
+
+    def log_captcha_failures_only?
+      return IdentityConfig.store.sign_in_recaptcha_log_failures_only
```

Comment:
Minor: Since the last statement of a method is returned implicitly, the `return` statement is unnecessary.

```suggestion
      IdentityConfig.store.sign_in_recaptcha_log_failures_only
```